Formula/aom.rb

```diff
@@ -0,0 +1,58 @@
+class Aom < Formula
+  desc "Codec library for encoding and decoding AV1 video streams"
+  homepage "https://aomedia.googlesource.com/aom"
+  url "https://aomedia.googlesource.com/aom", :using => :git, :tag => "v1.0.0", :revision => "cb108fb20fec79291588f4925499ac4cb3e64bfd"
```

Comment:
```
==> Checking out tag v1.0.0
HEAD is now at d14c5bb4f... [NORMATIVE]Allow non-identical sequence headers
Error: v1.0.0 tag should be cb108fb20fec79291588f4925499ac4cb3e64bfd
but is actually d14c5bb4f336ef1842046089849dee4a301fbbf0
```

Reply:
That was a stupid mistake on my part (copied the id of the parent commit).